### PR TITLE
fix: take destNamespace from user

### PR DESCRIPTION
**Pull request by SVKREP**
I am new to open source. I made the PR referring to this link: https://github.com/konveyor/move2kube/blob/main/contributing.md Please LMK if I need to correct something. 

This PR Solves: https://github.com/konveyor/move2kube/issues/930

I've built and tried Move2kube project. I am having trouble in testing this change. Can somebody suggest. Thanks!

### CODE REVIEW

File: transformer/kubernetes/apiresource/argocdapplication.go [hunk 1]
```diff
@@ -70,6 +74,18 @@ func (*ArgoCDApplication) createNewResource(irApplication irtypes.Application, t
 		clusterServer = deployToSameCluster
 	}
 	appGVK := v1alpha1.ApplicationSchemaGroupVersionKind
+	prob := qatypes.Problem{}
+	prob.ID = common.JoinQASubKeys(parameterizer.ParamQuesIDPrefix, `"`+appGVK.GroupVersion().String()+`"`, `"`+appGVK.Kind+`"`, "destNamespace")
+	prob.Type = qatypes.InputSolutionFormType
+	prob.Desc = "Enter destination namespace for argo cd pipeline"
+	ques, err := qaengine.FetchAnswer(prob)
+	if err != nil {
+		logrus.Errorf("failed to ask a question to the user in order to parameterize a k8s resource: %+v\nError: %q", prob, err)
```
Comment (HarikrishnanBalagopal) [inline transformer/kubernetes/apiresource/argocdapplication.go:83]: we need to return on error.

Don't use the `FetchAnswer` function use the `FetchStringAnswer` function https://github.com/konveyor/move2kube/blob/817b8c901e9d3622a7fcbf6db55a34a08427ca90/qaengine/engine.go#L210

Example: https://github.com/konveyor/move2kube/blob/817b8c901e9d3622a7fcbf6db55a34a08427ca90/transformer/kubernetes/apiresource/triggertemplate.go#L91C8-L97

File: transformer/kubernetes/apiresource/argocdapplication.go [hunk 2]
```diff
@@ -70,6 +74,18 @@ func (*ArgoCDApplication) createNewResource(irApplication irtypes.Application, t
...
+	}
+	destNamespace, ok := ques.Answer.(string)
+	if !ok {
+		logrus.Errorf("failed to ask a question to the user in order to parameterize a k8s resource: %+v\nError: the answer was not a string", prob)
```
Comment (HarikrishnanBalagopal) [inline transformer/kubernetes/apiresource/argocdapplication.go:87]: we need to return on error

**Review by HarikrishnanBalagopal — CHANGES_REQUESTED** [review]

**Comment by seshapad** [comment]
@SVKREP Please rebase your branch to have the latest upstream.

**Review by Akash-Nayak — CHANGES_REQUESTED** [review]
Hi @SVKREP, can you please make the changes requested by @HarikrishnanBalagopal on this PR-
https://github.com/konveyor/move2kube/pull/1069/files#r1281931155
https://github.com/konveyor/move2kube/pull/1069/files#r1281931265

**Comment by SVKREP** [comment]
Hi Team, @Akash-Nayak, @HarikrishnanBalagopal, @kmehant. Can someone please review this in your free time!

File: transformer/kubernetes/apiresource/argocdapplication.go [hunk 3]
```diff
@@ -70,6 +71,14 @@ func (*ArgoCDApplication) createNewResource(irApplication irtypes.Application, t
 		clusterServer = deployToSameCluster
 	}
 	appGVK := v1alpha1.ApplicationSchemaGroupVersionKind
+	ques := qaengine.FetchStringAnswer(
+		common.ConfigTransformersKubernetesArgoCDNamespaceKey,
+		"Enter destination namespace for argo cd pipeline",
```
Comment (Akash-Nayak) [inline transformer/kubernetes/apiresource/argocdapplication.go:76]: Can you please update the string to "Enter the destination namespace for the Argo CD pipeline"

File: transformer/kubernetes/apiresource/argocdapplication.go [hunk 4]
```diff
@@ -70,6 +71,14 @@ func (*ArgoCDApplication) createNewResource(irApplication irtypes.Application, t
...
+		[]string{"If this is not relevant to you then give an empty string to remove it from the YAML."},
```
Comment (Akash-Nayak) [inline transformer/kubernetes/apiresource/argocdapplication.go:77]: Can we also update the context to `[]string{If Argo CD pipeline is not relevant to you, then leave empty to use the default value for it.}`

**Comment by SVKREP** [comment]
@Akash-Nayak, I have made the changes, Thanks!

**Review by HarikrishnanBalagopal — APPROVED** [review]
LGTM

**Pull request merged**